lib/simple-connect-policy.js

```diff
@@ -6,9 +6,10 @@
 // anonymous handshake.
 //
 module.exports = class SimpleConnectPolicy {
-  constructor(login, password) {
+  constructor(login, password, version) {
```

Comment:
Why did you choose to provide version here instead of passing it to the `connect` method?